lib/routes/appwrite/blog.ts

```diff
@@ -0,0 +1,89 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import logger from '@/utils/logger';
+import puppeteer from '@/utils/puppeteer';
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['blog', 'reading'],
+    example: '/blog',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['appwrite.io/blog'],
+        },
+    ],
+    name: 'Appwrite Blog',
```

Comment:
Route name should not be the same as namespace name https://github.com/DIYgod/RSSHub/blob/bfc36a3c50d5da930260e3429234b991d8aae1ee/lib/routes/appwrite/namespace.ts#L4